src/commonmark/en/content/user/managing-dashboards.md

```diff
@@ -262,24 +268,31 @@ Click on the **Print** button in the upper right to trigger the browser print fu
 
 ![](resources/images/dashboard/dashboard-print-oipp.png)
 
-## Dashboard items with charts, pivot tables and maps
+## Dashboard items with charts, pivot tables or maps
+
+Dashboard items with charts, pivot table or maps may have a context menu button in the upper right corner of the item with additional viewing options, depending on the system settings that have been configured for the instance. If all the relevant system settings have been disabled, then there will not be a context menu button. Here are the possible menu options: 
 
 ### Switching between visualizations
 
 Dashboard items showing charts, pivot tables and maps can be toggled
-between these visualizations. Click on the item menu button in the upper 
-right corner of the item and choose the desired view:
+between these visualizations. Click on the item context menu button and choose the desired view (e.g., **View as Table**, **View as Map**, **View as Chart**):
 
 ![](resources/images/dashboard/dashboard-item-menu.png)
 
-> **Note**
-> 
->  This option may be disabled on your system if the system setting _Allow users to switch dashboard favorite view type_ is unchecked.)
 
-## Interpretations
+### View item in fullscreen
+
+To view the chart, table or map in fullscreen, click on the **View fullscreen** option.
```

Comment:
Should we mention how to get out of fullscreen?

Reply:
Added a sentence about that